va/va.h

```diff
@@ -2350,6 +2351,8 @@ typedef struct _VAPictureParameterBufferH264
         unsigned int value;
     } pic_fields;
     unsigned short frame_num;
+    unsigned char num_ref_idx_l0_default_active_minus1;
+    unsigned char num_ref_idx_l1_default_active_minus1;
```

Comment:
Agree with what xfengcarl said.
Based on H264 spec, the num_ref_idx_l0/l1_default_active_minus1 is the parameter that applies to the frame. Every slice can select the different num_ref_idx_l0/l1_active minus when the override flag exists. If the slice doesn't have the override flag, it can be derived from the num_ref_idx_l0/l1_default_active_minus1.